nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -88,6 +88,13 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         <IncludeCopyWinMDArtifactsOutputGroup>false</IncludeCopyWinMDArtifactsOutputGroup>
     </PropertyGroup>
 
+    <ItemDefinitionGroup>
+        <Midl Condition="'$(CppWinRTModernIDL)' != 'false'">
+            <AdditionalMetadataDirectories Condition="'%(AdditionalMetadataDirectories)' == '' And '$(WindowsSDK_MetadataFoundationPath)' != ''">$(WindowsSDK_MetadataFoundationPath);%(AdditionalMetadataDirectories)</AdditionalMetadataDirectories>
```

Comment:
Have you double-checked that this works as you expect? I think it will work if a project doesn't specify `AdditionalMetadataDirectories` for a given `Midl` item, but the old behavior (when it was in a target) was to prepend `$(WindowsSDK_MetadataFoundationPath)` unconditionally.

Reply:
I believe this is just there to handle a case where C++/WinRT is used outside of a UWP since Microsoft.Cpp.AppContainerApplication.props already sets this for a UWP.